test/exporter.js

```diff
@@ -133,7 +129,7 @@ test('Exporter.addCard with options (tags is string)', t => {
   t.is(cardsUpdate[':nid'], notesUpdate[':id'], 'should link both tables via the same note_id');
 });
 
-test('Exporter._getId', async t => {
+test('Exporter._getId', t => {
```

Comment:
Why was this change made?

Reply:
Ah I made that when I was debugging a test error. There was no need for it to be async, but it doesn't affect anything.